### ISSUE TITLE
Hide one pane (HTML/CSS/JS) but keep the other two?

### ISSUE THREAD

**Issue by fidian** [chinchang/web-maker#413]
Right now one can expand a single pane and collapse the other two. Is there a way to collapse one, giving up the space to the other two panes? I can manually drag and resize, but that's a problem when I want to move the space around.

I use Atomic CSS, so the CSS for my app is automatically generated. I don't ever need to worry about seeing CSS, which is super cool. My focus is on the HTML and JS. Sometimes I need to work more on the HTML side and others need a lot of attention on the JS pane. If I could collapse the CSS pane with a click, that would be wonderful. If I could then resize the JS or HTML frames by dragging the separator and it would keep the CSS pane small and adjust the sizes of the other two, that would be stellar.

**Comment by chinchang** [comment]
Definitely. Noted. Will see how this can be achieved in the UI. Suggestions welcome.
